cves/CVE-2006-5752.yml

```diff
@@ -48,14 +53,19 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
    - commit: a44b079370ea0d4fb5748dc510d9b14698a196cc
-     note:
+     note: specified a charset when setting content type 
    - commit: 6f0c8008746b58eb0a59f59501163eea2921d03d
-     note:
+     note: specified a charset when setting content type
 vccs:
   - commit: dba4852ebca38b6c58f7600e4ef0d57bd9c81a8f
-    note:
-  - commit:
-    note:
+    note:  
+  - commit: 15d69599aaf36251ee96f1bf6f8e6018b09be19a
+    note: in the history of the function from pointer to ap_rset_content_type
```

Comment:
This note may be more suited for putting in in the mistake section.

Reply:
moved to mistakes.